tables/Makefile.am

```diff
@@ -432,6 +432,12 @@ $(fuzzing_yml): $(srcdir)/Makefile.am
 
 all-local: $(fuzzing_yml)
 
+# tables that use macros need to be expanded
+BUILT_SOURCES = nl-chardefs.uti nl-NL-g0.utb
+CLEANFILES = nl-chardefs.uti nl-NL-g0.utb
+nl-chardefs.uti nl-NL-g0.utb: % : %.in Makefile
```

Comment:
That is a very interesting construct that I don't think I've seen before. Is that portable or just a GNU Make thing?

Reply:
Oh, no idea. I assumed it was a Make feature.

Reply:
I think it is a normal Make feature